xero_accounting.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: Xero Accounting API
-  version: "2.10.3"
+  version: "2.10.2"
```

Comment:
Should this be 2.10.4?

Reply:
Yes - i've got some dependent changes from a branch off 2.10.2
> https://github.com/XeroAPI/Xero-OpenAPI/pull/391

I'll remerge master on the OAS PR and get this fixed :P

Reply:
I think this is cached or something... pushed this change but not showing on github ui..

![Screen Shot 2021-03-25 at 4 37 26 PM](https://user-images.githubusercontent.com/2935387/112552417-638a2a80-8d88-11eb-8856-3ac9428ecf10.png)

Reply:
Ohh there it is..:) good catch!